articles/postgresql/flexible-server/how-to-connect-with-managed-identity.md

```diff
@@ -49,16 +49,26 @@ az ad sp list --display-name vm-name --query [*].appId --out tsv
 
 ## Create a PostgreSQL user for your Managed Identity
 
-Now, connect as the Microsoft Entra administrator user to your PostgreSQL database, and run the following SQL statements, replacing `CLIENT_ID` with the client ID you retrieved for your system-assigned managed identity:
+Now, connect as the Microsoft Entra administrator user to the database named `postgres` in the PostgreSQL databases, and run the following SQL statements, replacing `<identity_name>` with the name of the resources for which you created a system-assigned managed identity:
 
 ```sql
 select * from pgaadauth_create_principal('<identity_name>', false, false);
 ```
 
+Success looks like:
+```sql
+    pgaadauth_create_principal
+-----------------------------------
+ Created role for "<identity_name>"
+(1 row)
+```
+
 For more information on managing Microsoft Entra ID enabled database roles, see [how to manage Microsoft Entra ID enabled PostgreSQL roles](./how-to-manage-azure-ad-users.md)
 
 The managed identity now has access when authenticating with the identity name as a role name and the Microsoft Entra token as a password.
 
+> Note: if the managed identity is not valid, an error is returned: `ERROR:   Could not validate AAD user <ObjectId> because its name is not found in the tenant. [...]`
```

Comment:
```suggestion
> [!Note]
> If the managed identity is not valid, an error is returned: `ERROR:   Could not validate AAD user <ObjectId> because its name is not found in the tenant. [...]`.
```